main.py

```diff
@@ -81,6 +83,28 @@ async def get_experiment_info(file: str) -> Any:
     return remote.examine(file)
 
 
+@app.get("/experiment/submit/")
+async def submit_experiment(file: str, args: str = "{}") -> int:
+    """Submit the given experiment file.
+    
+    Args:
+        file: The path of the experiment file.
+        args: The arguments to submit which must be JSONifiable to a dictionary.
```

Comment:
Usually we say "JSON string", while "JSONifiable" refers to the objects which can be converted into a JSON, and JSON is a well-formatted _string_.
Therefore, "... must be a JSON string of a dictionary." would be more natural!

Reply:
Oh, thanks for letting me know! I thought I was confused.